### DISCUSSION TITLE
How can I use a comma within the switch case?

### DISCUSSION THREAD

**Discussion by artola** [ivanhofer/typesafe-i18n#548] [Q&A] [answered]
For the example below, in place of `An additional tax will be collected.` I would like to have `An additional tax, a few bucks for Uncle Sam, will be collected.`

```javascript
const translations: Translation {
   tax: 'Price: ${price:number}. {taxes|{yes: An additional tax will be collected. , no: No taxes apply.}}'
}

LL.tax({ price: '999', taxes: 'yes' })
// => 'Price: $999. An additional tax will be collected.'
LL.tax({ price: '99', taxes: 'no' })
// => 'Price: $99. No taxes apply.'
```

From example: https://github.com/ivanhofer/typesafe-i18n/tree/main/packages/runtime#switch-case

**Comment by ivanhofer** [comment]
Hi @artola,
this is currently not possible. I'll take a look if I can find a way to support that use case.

**Comment by ivanhofer** [comment, marked as answer]
You can update the latest version (`5.18.0`) and it should work like expected.

**Comment by artola** [comment]
@ivanhofer It is perfect. Thanks!
